src/components/OSCALControlProse.js

```diff
@@ -420,6 +419,10 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
         setParameter.values ??= [null];
         const setParameterRef = setParametersRefs[segment];
         // TODO - support for more than 1 item in values arrays
+
+        // Due to a lack of a good value for a key, the index is being used
+        // as a last resort.
+        /* eslint-disable react/no-array-index-key */
```

Comment:
I _really_ dislike these magic comment blocks of `eslint-disable` and a matching `eslint-enable` because it makes it too easy to disable the rule for the rest of the file. Instead, I think I'd like to see us just apply the lint to the single line where the linter complains. Generally the block style of these is best when you have a bunch of violations of a rule in a relatively small block. We have one violation in a big block.